docs/quickstart/ccd.md

```diff
@@ -310,3 +310,41 @@ If you've made mistakes and want to start over:
    (including the k8s namespace). Don't forget to recreate the
    namespace afterwards.
 
+[prometheus]: https://prometheus.io/
+[prom-helm]: https://github.com/prometheus-community/helm-charts
+[prom-sm]: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus-operator/servicemonitor.yaml
+
+## Monitoring the cluster via Prometheus
+Most components of STRM Privacy expose [Prometheus][prometheus] metrics and we've included the [Prometheus Cluster
+Operator][prom-helm] Helm chart. In order to define _scrape targets_ to Prometheus, we deploy  [Service
+Monitors][prom-sm] that expose the metrics endpoints in the Event Gateway.
+
+### the Prometheus server
+Access the Prometheus server either via [telepresence][telepresence] or via a port-forward
+
+    kubectl port-forward service/strmprivacy-kube-prometheu-prometheus 9090:9090&
```

Comment:
Typo in the service name.

Reply:
That is actually done like this by the prometheus operator, to keep the service name short enough :-)

Reply:
Fascinating 😅